website/docs/reference/warehouse-profiles/redshift-profile.md

```diff
@@ -63,7 +63,7 @@ my-redshift-db:
     dev:
       type: redshift
       method: iam
-      cluster_id: [cluster_id]
+      cluster_id: cluster_id
       host: hostname.region.redshift.amazonaws.com
       user: alice
       iam_profile: data_engineer # optional
```

Comment:
Below here, I also see 
>   db_groups: ['analysts']  

Should [`analysts`] become `ANALYSTS`  if we're standardizing on all caps for text that requires changing?